hack/release.toml

```diff
@@ -6,7 +6,7 @@ github_repo = "talos-systems/sidero"
 match_deps = "^github.com/(talos-systems/[a-zA-Z0-9-]+)$"
 
 # previous release
-previous = "v0.3.2"
+previous = "v0.4.1-beta.0"
```

Comment:
nit: I think it should be v0.4.0, the last tag which is shared on master and `release-0.4`

Reply:
rgr